docs/pages/guides/server-components.mdx

```diff
@@ -388,37 +327,112 @@ async function ExpensiveTask() {
+
+## Metadata
+
+React Server Components are a feature of React 19. To enable them, Expo CLI automatically uses a special canary build of React on all platforms. In the future, it will be removed when React 19 is enabled by default in React Native.
+
+As a result, you can use React 19 features such as placing `<meta>` tags anywhere in your app (web-only).
+
+```tsx app/index.tsx
 export default function Index() {
   return (
-    <View>
-      <Text>Hello, world!</Text>
-    </View>
+    <>
+      {process.env.EXPO_OS === 'web' && (
+        <>
+          <meta name="description" content="Hello, world!" />
+          <meta property="og:image" content="/og-image.png" />
+        </>
+      )}
+      <MyComponent />
+    </>
   );
 }
 ```
 
-- `render: 'static'` will render the component at build-time and never re-render it in production. This is similar to how classic static site generators work.
-- `render: 'dynamic'` will render the component at request-time and re-render it on every request. This is similar to how server-side rendering works.
+You can use this instead of the `Head` component from `expo-router/head`, but it only works on web for now.
 
-If you want client-side rendering, move your data fetching to a Client Component and control the rendering locally.
+## Known limitations
 
-Routes marked with `static` output will be rendered at build-time and embeded in the native binary. This enables rendering routes without making a server request (because the server request was made when the app was downloaded).
+> This is a very early technical preview that we're actively developing.
 
-The current default is `dynamic` rendering. In the future, we'll change the caching and optimizations to be smarter and more automatic.
+- Expo Snack does not support bundling Server Components.
+- EAS Update does not work with Server Components yet.
+- DOM components cannot use Server Actions yet.
+- Production deployment is very limited and not recommended yet.
+- Server Rendering RSC payloads to HTML is not supported yet. This means static and server output doesn't fully work yet.
+- `generateStaticParams` is not yet supported.
+- HTML `form` integration with Server Actions is not supported yet (this partially works automatically, but data is not encrypted).
+- `StyleSheet.create` and `Platform.OS` are not supported on native. Use standard objects for styles and `process.env.EXPO_OS` for platform detection.
+- Server Actions that invoke other server actions are not supported on Hermes due to a limitation in the Hermes runtime. This may be resolved with Static Hermes.
+
+## Full React Server Components mode
+
+> This mode is even more experimental!
+
+You can leverage even more features by enabling full React Server Components support, where the default rendering mode for routes is as server components instead of client components. This mode is still in development as the router and React Navigation need to be rewritten to support concurrency.
```

Comment:
```suggestion
Enabling full React Server Components support allows you to leverage even more features. In this mode, the default rendering mode for routes is server components instead of client components. It is still in development, as the Router and React Navigation need to be rewritten to support concurrency.
```

There is a typo "as" and let's improve the readability. Also, the router used here is the name of the library, so let's capitalize it.